.github/workflows/docker.yml

```diff
@@ -48,4 +48,3 @@ jobs:
           platforms: linux/amd64,linux/arm/v7
```

Comment:
can you please make this:

          platforms: linux/amd64,linux/arm64

?

armv7 is 32 bit

Reply:
https://docs.tendermint.com/master/nodes/running-in-production.html#validator-signing-on-32-bit-architectures-or-arm

Reply:
Thanks @faddat it was completely a typo. It updated now.